sdk/eventhub/azure-eventhubs/HISTORY.md

```diff
@@ -1,5 +1,26 @@
 # Release History
 
+## 5.0.0b2 (2019-08-06)
+
+**New features**
+
+- Added new class `EventDataBatch` for publication of a batch of events with known size constraint.
+- Added new method `create_batch` to producer for creating EventDataBatch objects.
```

Comment:
In Javascript, we combined the above 2 points as they are related

> Added method `create_batch()` on the `EventHubProducer` to create an `EventDataBatch` that can then be used to add events until the maximum size is reached. This batch object can then be used in the `send()` method to send all the added events to Event Hubs.

Then in a sub bullet we have the below:

> This allows publishers to build batches without the possibility of encountering the error around the message size exceeding the supported limit when sending events. It also allows publishers with bandwidth concerns to control the size of each batch published.

Reply:
thanks for the suggestion, doc is updated.